app/assets/stylesheets/full-width-pics.css

```diff
@@ -0,0 +1,120 @@
+/*!
+ * Start Bootstrap - Full Width Pics HTML Template (http://startbootstrap.com)
+ * Code licensed under the Apache License v2.0.
+ * For details, see http://www.apache.org/licenses/LICENSE-2.0.
+ */
+@import url(http://fonts.googleapis.com/css?family=Josefin+Sans:400,300,600);
+
+body {
+    margin-top: 50px; /* Required margin for .navbar-fixed-top. Remove if using .navbar-static-top. Change if height of navigation changes. */
+    background-color: #F8F8F8 ;
+}
+
+h1 {
+    font-family: 'Josefin Sans', sans-serif;
+}
+h2 {
+    font-family: 'Josefin Sans', sans-serif;
+}
+h3 {
+    font-family: 'Josefin Sans', sans-serif;
+}
+
+h4 {
+    font-family: 'Josefin Sans', sans-serif;
+}
+
+.nav {
+    font-family: 'Josefin Sans', sans-serif;
+}
+
+.image-bg-fluid-height,
+.image-bg-fixed-height {
+    text-align: center;
+    -webkit-background-size: cover;
+    -moz-background-size: cover;
+    background-size: cover;
+    -o-background-size: cover;
+}
+
+.image-bg-fluid-height {
+	background: url('../assets/china.png') no-repeat center center scroll;
```

Comment:
I'm pretty sure 2 tabs is standard for CSS, my editor doesn't seem to like anything less.